src/core/router.go

```diff
@@ -131,6 +132,7 @@ func initRouters() {
 	beego.Router("/service/notifications/jobs/adminjob/:id([0-9]+)", &admin.Handler{}, "post:HandleAdminJob")
 	beego.Router("/service/notifications/jobs/replication/:id([0-9]+)", &jobs.Handler{}, "post:HandleReplicationScheduleJob")
 	beego.Router("/service/notifications/jobs/replication/task/:id([0-9]+)", &jobs.Handler{}, "post:HandleReplicationTask")
+	beego.Router("/service/notifications/scheduler", &scheduler.Handler{}, "post:Handle")
```

Comment:
Is this the hook endpoint?



----

Better than `/api/schedules`?

POST `/api/schedules` for schedule 
DELETE `/api/schedules/:sid` for unschedule?